cmd/touch/touch.go

```diff
@@ -101,3 +102,33 @@ func Touch(ctx context.Context, fsrc fs.Fs, srcFileName string) (err error) {
 	}
 	return nil
 }
+
+// Recursively touch every file in f with time t
+func TouchRecursive(ctx context.Context, f fs.Fs, t time.Time) error {
+	timeAtr := time.Now()
+	if timeAsArgument != "" {
+		layout := defaultLayout
+		if len(timeAsArgument) == len(layoutDateWithTime) {
+			layout = layoutDateWithTime
+		} else if len(timeAsArgument) > len(layoutDateWithTime) {
+			layout = layoutDateWithTimeNano
+		}
+		var timeAtrFromFlags time.Time
+		if localTime {
+			timeAtrFromFlags, err = time.ParseInLocation(layout, timeAsArgument, time.Local)
```

Comment:
You must also declare `err`, e.g. `var err error` just below `var timeAtrFromFlags time.Time` above the if.

Reply:
@albertony Ah thank you for that, I have done that now.